tools/pot/openvino/tools/pot/configs/hardware/cpu.json

```diff
@@ -225,7 +225,47 @@
         {
             "type": "Concat",
             "attributes": {
-                "scales": "unified"
+                "unified_scales": true
+            }
+        },
+        {
+            "type": "LSTMCell",
+            "quantization": {
+                "activations": "q8_a",
+                "weights": "q8_w_sym"
+            },
+            "attributes": {
+                "unified_scales": [[0, 1], [3, 4]]
```

Comment:
I think we agreed on a different approach where these attributes are connected to the corresponding fields ("weights" or "activations") in the "quantization" section.